src/shared/utils/index.ts

```diff
@@ -21,9 +21,51 @@ import { RifRelayConfig } from '@rsksmart/rif-relay-light-sdk'
 import { ChainID, EOAWallet, OnRequest, WalletState } from 'lib/eoaWallet'
 import { RelayWallet } from 'lib/relayWallet'
 
+import { TokenSymbol } from 'screens/home/TokenImage'
+
 import { Wallet } from '../wallet'
 import { ErrorWithMessage } from '../types'
 
+export const rbtcMap = new Map([
+  [TokenSymbol.TRBTC, true],
+  [TokenSymbol.RBTC, true],
+  [undefined, false],
+])
```

Comment:
Looks like this was moved, right? So perhaps there was a reference to it that is now missing?

Reply:
Checked by the name, it's there